docs/writing-stories/mocking-providers.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Mocking providers
+---
+
+<!-- TODO: React & Solid only? -->
```

Comment:
Even if this is React and Solid specific, we should at least provide an intro and frame it accordingly wouldn't you agree?